mineral-ui/mineral-ui
# [Site] restructure site components to enable building guidelines pages

### Description
This PR is mostly structural changes and filler content to set up later development of Guidelines pages.

closes #225 

- created a `/packages/site/src/pages` folder that will hold some (all?) of the top-level content
- stopgap `SideNav.js` component

We can merge this into `master` and even publish the site if we so choose; the router has not been changed and the site still goes from `/` to `/components/button`.

### Motivation and context
We need to block out areas were we need content provided to us by other teams. In lieu of design comps, we can at least create the IA for the demo site and copy can be refined as we go along. Creating pages will get the conversation moving about what content needs to go where.

### Screenshots or videos, if appropriate
The extra pages are currently quite ugly, so I'm going to hide behind this door 🚪 

### How to test
1. fire up the site like normal
2. navigate to `/guidelines`
3. You should see a Table of Contents page. (do we need one? that's what we're here to figure out!). Other pages:
 - Color
 - Theming / Tokens
 - Typography
 - Getting Started
4. check out the `packages/site/src` folder structure. Is this what we want to have? I'm open to other better ideas.

### Types of changes
applicable. Note: Whatever you choose here should match your commit messages. -->
- New Feature

### Checklist




### How does this PR make you feel?

![there is no cow level](https://media.giphy.com/media/I6bnF1KUvMupa/giphy.gif)
